config_panel.json

```diff
@@ -36,10 +40,90 @@
                     "id": "users",
                     "options": [
                         {
-                            "name": "Authorized external user creation ?",
+                            "name": "use_web_account",
+                            "ask": {
+                                "en": "Authorized external user creation ?"
+                            },
                             "help": "Allow user to be created without yunohost account.",
-                            "id": "use_web_account",
-                            "type": "bool",
+                            "type": "boolean",
+                            "default": true
+                        }
+                    ]
+                },
+                {
+                    "name": "Backup strategy",
+                    "id": "backup_strategy",
```

Comment:
Somewhere in that category, you should add a BIG warning explaining that every part you'll not backup would be lost in case of restore.

Reply:
I can't display message for all this section (or I missed something?). Should I duplicate the warning?...

Reply:
In fact, after some tests, the "help" of the section is displayed, but not the help inside each options. For me the "help" of the section should be called "description", and help of all options should be displayed on the same row of this one

Reply:
![Capture du 2019-09-02 12-41-47](https://user-images.githubusercontent.com/3713691/64109174-121de300-cd7f-11e9-9229-7981fdd5b91a.png)
I have something like that for now

Reply:
I don't think you can manage the size of the warning, but at least it's for the whole category.
Didn't understood (quickly read) your thing about help and description, but as this feature is still experimental, don't hesitate to tell @Psycojoker if anything could be enhanced.